__tests__/screens/BackupWords.test.tsx

```
@@ -0,0 +1,220 @@
+      // Subsequent calls: shuffling word options for display.
+      // Return as-is (no shuffle) so we can predict button order.
+      return arr;
+    }),
+  };
+});
+
+// Mock wallet-lib (needed by transitive imports)
+jest.mock('@hathor/wallet-lib', () => {
+  class MemoryStore {
+    getItem() { return null; }
+    setItem() {}
```

ESLint

[error] Expected blank line between class members.

(lines-between-class-members)